### PR TITLE
Bug fix: Secondary Logformat entries are not shown when the access log format is json

### PR DESCRIPTION
**Pull request by VirajSalaka**
### Issues

Fixes #

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments

Locally tested

---
#### Maintainers: Check before merge
- [ ] Assigned 'Type' label
- [ ] Assigned the project
- [ ] Validated respective github issues
- [ ] Assigned milestone to the github issue(s)

### CODE REVIEW

File: adapter/internal/oasparser/envoyconf/access_loggers.go [hunk 1]
```diff
@@ -78,6 +78,13 @@ func getFileAccessLogConfigs() *config_access_logv3.AccessLog {
 		for k, v := range logConf.AccessLogs.JSONFormat {
 			logFields[k] = &structpb.Value{Kind: &structpb.Value_StringValue{StringValue: v}}
 		}
+		formattedSecondaryLogFormat := strings.ReplaceAll(logConf.AccessLogs.SecondaryLogFormat, "'", "")
```
Comment (renuka-fernando) [inline adapter/internal/oasparser/envoyconf/access_loggers.go:81]: Shall we do the other way around?
Let's remove `SecondaryLogFormat` and read it from a new config `JSONFormatSecondary`. We can use the value of the map for Text format if needed.

File: adapter/internal/oasparser/envoyconf/access_loggers.go [hunk 2]
```diff
@@ -78,6 +78,13 @@ func getFileAccessLogConfigs() *config_access_logv3.AccessLog {
...
+		secondaryLogFields := strings.Split(formattedSecondaryLogFormat, " ")
+		// iterate through secondary log fields and update the logfields map
+		for _, field := range secondaryLogFields {
+			logFields[strings.ReplaceAll(strings.TrimSpace(field), "%", "")] =
```
Comment (slahirucd7) [inline adapter/internal/oasparser/envoyconf/access_loggers.go:85]: Do we need to use `TrimSpace` again ?

Comment (VirajSalaka) [reply adapter/internal/oasparser/envoyconf/access_loggers.go:85]: ah nope
will fix

**Review by renuka-fernando — APPROVED** [review]

**Review by renuka-fernando — DISMISSED** [review]

**Review by slahirucd7 — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[succeeded]  : dev-deployment-v2 : 20240326.5

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : dev-deployment-v2 : 20240326.5

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(EastUS) cluster : dev-deployment-v2 : 20240326.5

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : dev-deployment-v2 : 20240326.5

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : stage-deployment-v2 : 20240327.2

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(CDP2-WU2) cluster : stage-deployment-v2 : 20240327.2

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : stage-deployment-v2 : 20240327.2

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(EastUS) cluster : stage-deployment-v2 : 20240327.2

**Comment by choreo-cicd** [comment]
[failed] Controlplane cluster : prod-deployment-v2 : 20240328.5

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(CDP2-WU2) cluster : prod-deployment-v2 : 20240328.5

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : prod-deployment-v2 : 20240328.5

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(EastUS) cluster : prod-deployment-v2 : 20240328.5

**Comment by choreo-cicd** [comment]
[] Dataplane(EastUS) cluster : prod-deployment-v2 : 20240328.5

**Comment by choreo-cicd** [comment]
[] Controlplane cluster : prod-deployment-v2 : 20240328.5

**Comment by choreo-cicd** [comment]
[] Dataplane(CDP2-WU2) cluster : prod-deployment-v2 : 20240328.5

**Comment by choreo-cicd** [comment]
[] Dataplane(NorthEU) cluster : prod-deployment-v2 : 20240328.5

**Pull request merged**